content/blog/2018/07/2018-07-19-jenkins-x-accelerate.adoc

```diff
@@ -0,0 +1,159 @@
+a staging environment to be held there for a short amount of time before being promoted, ideally in an automated fashion.
+
+The default Jenkins X pipelines provide deployment automation via environments.  These are customisable to suite your own 
+CI / CD pipeline requirements.
+
+Jenkins X recommends Staging should act as a near as possible reflection on production, ideally with real production data 
+shadowed to it using a service mesh to understand the behaviour.  This also helps when developing changes in preview where we 
+can link to non production services in staging.
+
+_Related Accelerate capability: Automate your deployment process_
+
+=== Use trunk-based development
+
+The Accelerate book found that teams which use trunk based development with short lived branches performed better.  This has 
+always worked for the Jenkins X core team members so this was an easy capability for Jenkins X to implement when setting up 
+Git repositories and CI/CD jobs.
+
+=== Implement Continuous Integration
+
+Jenkins X sees CI as the effort of validating a proposed change via pull requests before it is merged to master.  Jenkins X 
+will automatically configure source code repositories, Jenkins and Kubernetes to provide Continuous Integration of the box.
+
+=== Implement Continuous Delivery
+
+Jenkins X sees CD as the effort of taking that change after it's been merged to master through to running in a live 
+environment.  Jenkins X automates many parts in a release pipeline:
+
+Jenkins X advocates the use of semantic versioning.  We use git tags to calculate the next release version which means we 
+don't need to store the latest release version in the master branch.  Where release systems do store the last or next version 
+in Git repos it means CD becomes hard, as a commit in a release pipeline back to master triggers a new release.  This results 
+in a recursive release trigger.  Using a Git tag helps avoid this situation which Jenkins X completely automates.
+
+Jenkins X will automatically create a released version on __every__ merge to master which can then potentially progress 
+through to production.
+
+=== Use loosely coupled architecture
+
+By targeting Kubernetes users of Jenkins X can take advantage of many of the cloud features that help design and develop 
+loosely coupled solutions.  Service discovery, fault tolerance, scalability, health checks, rolling upgrades, container 
+scheduling and orchestration to name just a few examples of where Kubernetes helps.
+
+=== Architect for empowered teams
+
+Jenkins X aims to help polyglot application developers.  Right now Jenkins X has quickstarts and automated CI/CD setup with 
+language detection for Golang, Java, NodeJS, .Net, React, Angular, Rust, Swift and more to come.  What this also does is 
+provide a consistent Way Of Working so developers can concentrate on developing.
```

Comment:
Is "Way Of Working" supposed to be capitalized?

Reply:
It is intentional yes, the top google hit I get is this article which capitalises it http://www.newwaysofworking.org.uk/the-new-way-of-working/ though actually let me lowercase the "Of"

Reply:
fixed